portality/static/widget/simple_search_body.html

```diff
@@ -1,12 +1,35 @@
-<link rel="stylesheet" href="https://doaj.org/static/widget/simple_search_widget.css">
-
-<form id="doaj-search-form" name="topsearch" action="https://doaj.org/search" method="POST">
-    <input type="hidden" name="ref" value="ssw">
-    <div class="control-group" id="doaj-search-container">
-        <a href="https://doaj.org/" target="_blank"><img id="doaj-simple-search-logo" src="https://doaj.org/static/doaj/images/logo_cropped.jpg" alt="Directory of Open Access Journals" title="Directory of Open Access Journals"></a>
-        <div>
-            <input type="text" name="q" placeholder="Search the DOAJ" id="doaj-search-input"/><span id="doaj-search-button-container"><button type="submit" formtarget="_blank" id="doaj-search-button">Search</button></span>
+<div class="modal__dialog" role="document">
```

Comment:
It’s not a modal, so we should not use this class (also with BEM, we should not be using element classes, `modal__dialog` without their corresponding blocks, `modal`).